readme.md

```diff
@@ -450,17 +450,27 @@ Object whose `chain` doesn't transform the contents. See [Monad](#monad)
 An object that implements a `map` function that takes a function which is run on the contents of that object. A functor must adhere to two rules:
 
 __Preserves identity__
-```
-object.map(x => x) ≍ object
-```
+
+```js
+object.map(x => x)
```

Comment:
Wondering if we demonstrate a simple `map` implementation or maybe use `array` rather?

Reply:
We got map defined for `Maybe` near the bottom.

Reply:
I updated to use the Option implementation. That better?